spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/config/MetricsReplicationEnvironmentPostProcessorTests.java

```diff
@@ -154,71 +150,42 @@ void wavefrontPropertiesReplicationWithPlaceholders() {
-				// Inherited property from parent PushRegistryProperties
-				"--management.metrics.export.wavefront.batch-size=20000")) {
-			for (String commonPropPrefix : COMMON_APPLICATION_PREFIXES) {
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.wavefront.enabled", "true");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.wavefront.enabled", "true");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.wavefront.api-token", "654-token");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.wavefront.source", "my-source");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.wavefront.batch-size", "20000");
-			}
-			assertEnvHasProperty(ctx, COMMON_STREAM_PROP_PREFIX + "management.metrics.export.wavefront.uri", "https://StreamUri");
-			assertEnvHasProperty(ctx, COMMON_TASK_PROP_PREFIX + "management.metrics.export.wavefront.uri", "https://TaskUri");
-			// Boot3 variants are also not overridden
-			assertEnvHasProperty(ctx, COMMON_STREAM_PROP_PREFIX + "management.wavefront.metrics.export.uri", "https://StreamUri");
-			assertEnvHasProperty(ctx, COMMON_TASK_PROP_PREFIX + "management.wavefront.metrics.export.uri", "https://TaskUri");
+				"--management.wavefront.metrics.export.enabled=true",
+				"--management.wavefront.api-token=654-token",
+				"--%smanagement.wavefront.uri=https://StreamUri".formatted(COMMON_STREAM_PROP_PREFIX),
+				"--%smanagement.wavefront.uri=https://TaskUri".formatted(COMMON_TASK_PROP_PREFIX),
+				"--management.wavefront.uri=https://vmware.wavefront.com")) {
+			assertEnvHasProperty(ctx, COMMON_STREAM_PROP_PREFIX + "management.wavefront.uri", "https://StreamUri");
+			assertEnvHasProperty(ctx, COMMON_TASK_PROP_PREFIX + "management.wavefront.uri", "https://TaskUri");
 		}
 	}
 
 	@Test
 	void influxPropertiesReplication() {
 		try (ConfigurableApplicationContext ctx = applicationContext(
-				"--management.metrics.export.influx.enabled=true",
-				"--management.metrics.export.influx.db=myinfluxdb",
-				"--management.metrics.export.influx.uri=http://influxdb:8086",
-				// Inherited property
-				"--management.metrics.export.influx.batch-size=20000")) {
+				"--management.influx.metrics.export.enabled=true",
+				"--management.influx.metrics.export.db=myinfluxdb",
+				"--management.influx.metrics.export.uri=http://influxdb:8086",
+				"--management.influx.metrics.export.batch-size=20000")) {
 			for (String commonPropPrefix : COMMON_APPLICATION_PREFIXES) {
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.influx.enabled", "true");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.influx.db", "myinfluxdb");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.influx.uri", "http://influxdb:8086");
-				assertEnvHasProperty(ctx, commonPropPrefix + "management.metrics.export.influx.batch-size", "20000");
```

Comment:
Just curious as to why we removed batch-size from the assert tests?

Reply:
They are still there - they are hidden below